Pokemon Unity/Assets/Scripts2/Saving/GlobalSaveManager.cs

```diff
@@ -0,0 +1,288 @@
+﻿using System.Runtime.Serialization.Formatters.Binary;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System;
+using UnityEngine.SceneManagement;
```

Comment:
Still being attached to unity here. Is this class unity dependant?

Reply:
If this is a "global" class manager, please remove it from unity. Let it work in background. It doesn't need to inherit unity for "saving to text" file.